sky/packages/sky/lib/src/fn3/binding.dart

```diff
@@ -15,7 +15,7 @@ class WidgetFlutterBinding extends FlutterBinding {
   }
 
   /// Ensures that there is a FlutterBinding object instantiated.
-  static void initBinding() {
+  static void _ensureInitialized() {
```

Comment:
Why private?
How do you make sure you get a WidgetFlutterBinding if this is private, if you're not using runApp()? (For example, spinning_mixed.)